subsonic/api.go

```diff
@@ -12,6 +12,7 @@ import (
 	"strings"
 
 	"github.com/spezifisch/stmps/logger"
+	"github.com/spf13/viper"
 )
 
 type SubsonicConnection struct {
```

Comment:
Configurations like "number of random songs to add" of this subsonic package should ideally come through this (SubsonicConnection) struct. So we need a variable here that's filled by the function instantiating us.